addons/packages/vsphere-cpi/1.20.0/bundle/config/schema.yaml

```diff
@@ -0,0 +1,79 @@
+#! schema.yaml
+
+#@data/values-schema
+#@schema/desc "OpenAPIv3 Schema for vsphere-cpi"
+---
+vsphereCPI:
+  #@schema/desc "The cryptographic thumbprint of the vSphere endpoint's certificate"
+  tlsThumbprint: ""
+  #@schema/desc "The IP address or FQDN of the vSphere endpoint"
+  #@schema/nullable
+  server: 10.1.1.1
+  #@schema/desc "The datacenter in which VMs are created/located"
+  #@schema/nullable
+  datacenter: ""
+  #@schema/desc "Username used to access a vSphere endpoint"
+  #@schema/nullable
+  username: ""
+  #@schema/desc "Password used to access a vSphere endpoint"
+  #@schema/nullable
+  password: ""
+  #@schema/desc "The region used by vSphere multi-AZ feature"
+  #@schema/nullable
+  region: ""
+  #@schema/desc "The zone used by vSphere multi-AZ feature"
+  #@schema/nullable
+  zone: ""
+  #@schema/desc "The flag that disables TLS peer verification"
+  insecureFlag: False
+  #@schema/desc "The IP family configuration"
+  #@schema/nullable
+  ipFamily: ""
+  #@schema/desc "Internal VM network name"
+  #@schema/nullable
+  vmInternalNetwork: ""
+  #@schema/desc "External VM network name"
+  #@schema/nullable
+  vmExternalNetwork: ""
+  cloudProviderExtraArgs:
+    #@schema/desc "External arguments for cloud provider"
+    tls-cipher-suites: "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384"
+  nsxt:
+    #@schema/desc "A flag that enables pod routing"
+    podRoutingEnabled: false
```

Comment:
this is optional

Reply:
Yes, they are optional, but not nullable